docs/en/installation-guide/database-initialization.rst

```diff
@@ -71,17 +71,28 @@ There are now 6 `schemas <https://teksi.github.io/wastewater/en/user-guide/layer
 + tww_vl
 
 
-Create  minimal roles and access
+Create minimal roles and access
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There are four default roles in TEKSI databases
 
-.. note:: The TWW roles are defined in the
-+ `12_0_roles.sql <https://github.com/TWW/datamodel/blob/master/12_0_roles.sql>`_ (per cluster) and
-+ `12_1_roles.sql <https://github.com/TWW/datamodel/blob/master/12_1_roles.sql>`_ (per database) files.
+- Viewer: Can consult tables and views.
+- User: Can edit data.
+- Manager: Can edit data and value lists.
+- Admin: Database administrator.
 
+.. note:: The TWW roles are defined in the python script https://github.com/TWW/datamodel/blob/master/create_roles.py. The script requires the pgserviceparser package from OpenGIS https://github.com/opengisch/pgserviceparser/.
 
-It is recommended to use these when using TWW in a production environment.
+You can call the script from the command line with the following arguments:
+
+* ´-m´ or ´--modelname´: Abbreviation of the datamodel (here: tww)
```

Comment:
module name ?

Reply:
These changes are pending, I first need the OK of the functionality before altering the docs again